src/content/en/fundamentals/getting-started/primers/async-functions.md

```diff
@@ -0,0 +1,249 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: TODO
+
+{# wf_published_on: 2016-10-07 #}
+{# wf_updated_on: 2016-10-07 #}
+
+# Async functions - making promises friendly {: .page-title }
+
+{% include "web/_shared/contributors/jakearchibald.html" %}
+
+Async functions are enabled by default in Chrome 55 and they're quite frankly marvelous. They allow you to write promise-based code as if it were synchronous, but without blocking the main thread.
+
+Async functions look like this:
+
+    async function myFirstAsyncFucntion() {
+      try {
+        const resolvedValue = await aPromise;
+      }
+      catch (rejectedValue) {
+        // …
+      }
```

Comment:
so because async functions always return a promise, you don't explicitly need a return val? Might want to mention that.